instrumentation/opentelemetry-instrumentation-asgi/tests/test_asgi_middleware.py

```diff
@@ -135,7 +135,7 @@ def validate_outputs(self, outputs, error=None, modifiers=None):
             },
             {
                 "name": "GET asgi",
-                "kind": trace_api.SpanKind.SERVER,
+                "kind": trace_api.SpanKind.INTERNAL,
```

Comment:
Shouldn't this ASGI span be of kind SERVER? Is there another span wrapping it in the context?

Reply:
Yes it should be, thank you